roles/create-ai-environment/tasks/main.yaml

```diff
@@ -0,0 +1,289 @@
+- name: Set Default Hardware type Parameters
+  set_fact:
+    user_file: "user_data"
+  when: not is_gpu_enabled
+
+- name: Split Hardware type Parameters for GPU
```

Comment:
Can these code of set/split hardware type para into another role, seems it is used several times.

Reply:
pre-run will be removed as well.